To-do list/Controllers/TaskDetailsViewController.swift

```diff
@@ -7,39 +7,44 @@
 
 import UIKit
 
+protocol TaskDetailsViewControllerDelegate: AnyObject {
+    func sendTaskDetails(_: TaskDetailsViewController, didCreateUpdate task: Task)
+}
+
 class TaskDetailsViewController: UIViewController {
     
+    weak var delegate: TaskDetailsViewControllerDelegate?
+    
     @IBOutlet weak var titleTextField: UITextField!
     @IBOutlet weak var descriptionTextField: UITextField!
     @IBOutlet weak var statusButtonOutlet: UIButton!
     
-    var users = [User]()
-    var userIndex = Int()
-    var taskIndex = Int()
+    
+
+    var task = Task()
     var counter = 0
-    var status = "to do"
-    var titleField = ""
+    var status = Task.Status.todo
```

Comment:
task's status should not be declared separately from task. Delete this var and use 'task.status' where needed.

Reply:
done, var deleted